website/static/js/contribManager.js

```diff
@@ -391,19 +426,38 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
         });
     };
 
+    self.afterRender = function(elements, data) {
+        var table;
+        if (data === 'contrib') {
+            table = self.table[0];
+        }
+        else if (data === 'admin') {
+            table = self.adminTable[0];
+        }
+        if (!!table) {
+            rt.responsiveTable(table);
+        }
+    };
+
+    self.collapsed = ko.observable(true);
+
+    self.checkWindowWidth = function() {
```

Comment:
This is a bit of a misnomer; "check" implies that there are no side effects, but calling this sets `self.collapsed`. Maybe rename to `resetWindowWidth`?